waspc/src/Wasp/Generator/Job/Common.hs

```diff
@@ -0,0 +1,24 @@
+module Wasp.Generator.Job.Common
+  ( getJobMessageOutHandle,
+    getJobMessageContent,
+  )
+where
+
+import qualified Data.Text as T
+import System.Exit (ExitCode (..))
+import System.IO (Handle, stderr, stdout)
+import qualified Wasp.Generator.Job as J
+
+getJobMessageOutHandle :: J.JobMessage -> Handle
```

Comment:
Tricky call. I think both are valid. I would say here slightly since they are more specific utility than just operating on the types themselves as generic helpers.